plugins/src/android/OptimoveSDKPlugin.java

```diff
@@ -9,8 +11,167 @@
 
 public class OptimoveSDKPlugin extends CordovaPlugin {
 
+    private static final String SET_USER_ID = "setUserId";
+    private static final String SET_USER_EMAIL = "setUserEmail";
+    private static final String REPORT_EVENT = "reportEvent";
+    private static final String REPORT_SCREEN_VISIT = "reportScreenVisit";
+    private static final String REGISTER_USER = "registerUser";
+    private static final String GET_VISITOR_ID = "getVisitorId";
+    private static final String GET_CURRENT_USER_IDENTIFIER = "getCurrentUserIdentifier";
+
     @Override
     public boolean execute(String action, JSONArray args, CallbackContext callbackContext) throws JSONException {
+        switch (action) {
+            case SET_USER_ID:
+                String userId = args.getString(0);
```

Comment:
General style question on these: is it better to do the logic of unpacking arguments (and in some others, branching), in the switch case, or better to pass through args to the helper method (more similar to Kumulos plugin)?